airflow/models/dagbag.py

```diff
@@ -391,6 +396,14 @@ def collect_dags(
             'dagbag_import_errors', len(self.import_errors), 1)
         self.dagbag_stats = sorted(
             stats, key=lambda x: x.duration, reverse=True)
+        for file_stat in self.dagbag_stats:
+            dag_ids= dags_by_name[file_stat.dags]
+            if file_stat.dag_num >= 1:
+                # if we found multiple dags per file, the stat is 'dag_id1 | dag_id2'
```

Comment:
Hmm I didn't find such support in https://statsd.readthedocs.io/en/v3.2.1/index.html

Reply:
which support? '|' in metric name?

Reply:
Yup

Reply:
I didn't find it no :) But I change it to use underscore.

Reply:
If we use underscore, maybe the comment line requires update? Or we can also consider removing it.